src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -713,6 +721,27 @@ public boolean isAllFeedbackPathOptionsEnabled() {
         return true;
     }
     
+    public boolean isAllRecipientOptionsDisplayed(int questionNumber) {
+        List<WebElement> recipientOptions =
+                browser.driver.findElements(By.cssSelector("#recipienttype-" + questionNumber + " option"));
+        for (WebElement recipientOption : recipientOptions) {
+            String recipientOptionStyle = recipientOption.getAttribute("style");
+            if ("display: none;".equals(recipientOptionStyle)) {
```

Comment:
I had initially tried `isDisplayed()` however it `returns true` whether the option has `display: none` or not. It might be because option tags are treated differently, I'm still googling around for the reason